WooCommerce/Classes/POS/Controllers/PointOfSaleOrderController.swift

```diff
@@ -174,6 +190,32 @@ private extension PointOfSaleOrderController {
     }
 }
 
+@available(iOS 17.0, *)
+private extension PointOfSaleOrderController {
+    @MainActor
+    func isPluginSupported(_ pluginName: String, minimumVersion: String, siteID: Int64) async -> Bool {
```

Comment:
Ideally, this would all be isolated within `POSReceiptService`, but as I understand, we cannot check for feature flags in Yosemite?

Reply:
Yup the feature flag check is in the app layer. Moreover, `VersionHelpers` is needed to check the version requirement and right now it's in the app layer as well (I think this could be moved to the Yosemite, but thought to keep the changes small).